plugins/out_azure/azure_conf.c

```diff
@@ -61,7 +64,6 @@ struct flb_azure *flb_azure_conf_create(struct flb_output_instance *ins,
     }
     else {
         flb_plg_error(ctx->ins, "property 'shared_key' is not defined");
-        flb_azure_conf_destroy(ctx);
```

Comment:
This fix seems to be wrong. If you just remove this line, it will cause
memleak since no one else would deallocate `ctx`.

You need to study the code to make sure that 1) everything is deallocated
properly on every path and 2) you never free the same thing twice